packages/api/src/models/medical/patient.ts

```diff
@@ -18,6 +19,10 @@ export class PatientModel extends BaseModel<PatientModel> implements Patient {
         facilityIds: {
           type: DataTypes.ARRAY(DataTypes.STRING),
         },
+        externalId: {
+          type: DataTypes.STRING,
+          allowNull: true,
```

Comment:
nit: we only need this on the table declaration, here its mostly so Sequelize know what type to expect from the DB when converting to the model's TS type, AFAIK